test/evergreen.yml

```diff
@@ -2538,11 +2538,9 @@ tasks:
   # End of Python unit test tasks
 
   - name: s-all
-    tags: ["pull_request"]
```

Comment:
Will this change remove this task from the PR testing scope?

Reply:
Could we update the PR testing to take the new tag?

Reply:
Sure, added